scivision_test_plugin/model.py

```diff
@@ -15,13 +22,29 @@ def __init__(self):
         pass
 
     def do_something(
-        self,
-        preserve_range: bool = True,
-        sigma: float = 10.0,
-        image: np.ndarray = np.empty((64, 64, 1)),
+            self,
+            preserve_range: bool = True,
+            sigma: float = 10.0,
+            image: np.ndarray = np.empty((64, 64, 1)),
     ) -> np.ndarray:
         return gaussian(image, sigma=sigma, preserve_range=preserve_range)
 
 
+class ImageNetModel:
+    def __init__(self):
+        self.pretrainedModel, self.preprocess_input = Classifiers.get('resnet18')
+
+    def predict(self, image: np.ndarray) -> np.ndarray:
+        model = self.pretrainedModel(
+            input_shape=(224, 224, 3), weights="imagenet", classes=1000
+        )
```

Comment:
We shouldn't need to build the model each time?

```suggestion
```

Reply:
so for the prediction function, do we have to change it to self.pretrainedModel.predict() instead of model.predict(), right?